src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -190,6 +191,33 @@ private static String getMessageForConstraintViolation(final ConstraintViolation
     }
   }
 
+  /**
+   * Return collection exercises for each of the the surveys
+   * in the given list of survey ids. Return data as a Json dictionary.
+   *
+   * @param surveyIds survey Ids for which to get collection exercises
+   * @param liveOnly Boolean , if set only returns live collection exercises
+   * @return json dictionary or collection exercises per survey
+   * @throws CTPException on resource not found
+   */
+  @RequestMapping(value = "/surveys", method = RequestMethod.GET, produces = "application/json")
+  public ResponseEntity<HashMap> getCollectionExercisesForSurveys(
+      final @RequestParam List<UUID> surveyIds,
+      @RequestParam("liveOnly") Optional<Boolean> liveOnly) {
+
+    HashMap<UUID, List<CollectionExercise>> surveyCollexDict = null;
```

Comment:
You don't need to set this to `null`, it inherently is until you instantiate it. Also :art: but I would name this `surveyCollexMap` given there's no such thing as a Java `dict`